### ISSUE TITLE
Find a new home for "Terms"

### ISSUE THREAD

**Issue by jthiller** [helium/docs#1750]
Pulled from the governance page.

```## Terms

- **Wallet** - A digital application designed for interacting with blockchain-based applications.
  Wallets enable users to store and manage their digital assets, such as digital tokens and NFTs.
- **Blockchain** - A shared, immutable ledger that facilitates the process of recording transactions
  and tracking assets in a business network. An asset can be tangible (a house, car, cash, land) or
  intangible (intellectual property, patents, copyrights, branding). Virtually anything of value can
  be tracked and traded on a blockchain network, reducing risk and cutting costs for all
  involved.[^1]
- **Solana** - A blockchain that uses a proof-of-stake ("PoS") mechanism to provide smart contract
  functionality as well as a mechanism called proof-of-history ("PoH"), which uses hashed timestamps
  to verify when transactions occur.
- **Token** - Tokens are digital assets that utilize cryptography for secure transactions, represent
  value, and can be used for various purposes within a decentralized network or ecosystem.
- **Non-Fungible Token ("NFT")** - A unique digital identifier that is recorded on a blockchain and
  is used to certify ownership and authenticity. It cannot be copied, substituted, or subdivided.
- **Consensus Protocols** – A set of rules that define the instructions for how to be an autonomous
  participant in a distributed computer network and come to an agreement about the exact state of
  common data.
- **Node** - Refers to individual devices or computers that participate in maintaining the network
  by storing a copy of the blockchain, validating transactions, and relaying information to achieve
  consensus.
- **Validators** - A distributed network actor responsible for verifying and validating transactions
  and blocks, maintaining the network's security and consensus.
- **Proof-of-Stake ("PoS")** - A PoS system is a type of blockchain consensus protocol that achieves
  verification by selecting validators in proportion to their quantity of holdings in the associated
  cryptocurrency.
- **Proof-of-History ("PoH")** - A consensus protocol unique to Solana, Proof-of-History is a
  verifiable and sequential timestamping mechanism that provides a trustless and efficient ordering
  of events in the blockchain.
- **Helium** - A global decentralized wireless network of networks utilizing a token-incentive
  model.
- **Helium Improvement Proposals ("HIP")** - A bill for changes to the network.
- **Discord** - A web-based community chat platform.
- **GitHub** - A web-based platform that allows developers to collaborate, manage, and store their
  code repositories.
- **Repository** - A storage location that contains all the files, documentation, and version
  history of a project.
- **Realms** - A platform for decentralized projects to manage token-based governance operations and
  manage token-based governance operations.
- **Helium Network Token ("HNT")** - The native token of the Helium Network, used for incentivizing
  and securing the decentralized wireless network of networks through redemption and swap features.
- **IOT subNetwork Token ("IOT")** - A token specific to the LoRaWAN Network, used for incentivizing
  and securing the decentralized wireless network for IoT devices.
- **MOBILE subNetwork ("MOBILE")** - A token specific for the 5G CBRS Network, used for
  incentivizing and securing the decentralized wireless network for Cellular/Mobile devices.
- **Staking (or “Lock Up”)** - The act of temporarily locking up tokens as collateral to participate
  in block validation and consensus mechanisms, earning rewards for supporting network operations or
  generating Voting Power.
- **Voting Power** - 100 veHNT is more than ten veHNT. The higher amount would mean you have more
  weight in deciding vote outcomes. The more HNT you stake for veHNT, the more voting power you
  have.
- **Vote Escrowed ("ve")** - A position created by staking your tokens that only functions for
  Network or subNetwork voting. Vote Escrowed positions show up in wallets as NFTs and are
  non-transferrable.
- **veHNT** - A position created by staking your HNT. Used for voting on proposals that affect the
  entire Helium Network of Networks model.
- **veIOT** - A position created by staking your IOT. Used for IOT Network governance.
- **veMOBILE** - A position created by staking your MOBILE. Used for MOBILE Network governance.
```

**Comment by Dumpling-girl** [comment]
Would you like this on the FAQ page as a sub heading titled 'Terminology' or a separate heading placed under 'History of Helium'.
I also saw on the main landing page some terminology explained. Some are not the same as in this list from the governance section. Maybe we should harmonize terminology first before making a page?

**Comment by jthiller** [comment]
Yes big +1 on harmonizing terminology.
Alongside an FAQ page makes sense.

**Comment by Dumpling-girl** [comment]
Ok, I will make a PR suggestion for harmonization and a new page, tomorrow.

**Comment by Dumpling-girl** [comment]
I  made a gist, but I haven't figured out yet how to get this into GitHub Desktop and do a PR.
Could you look at it please and provide feedback if you have time?
https://gist.github.com/Dumpling-girl/3f368422269507f94aa00009d4ccbe89

**Comment by Dumpling-girl** [comment]
@jthiller Ok, I figured out how to add new file XD
PR #1791.

**Comment by jthiller** [comment]
Closed in #1791
